docusaurus/docs/reactnative/additional_resources.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: additional-resources
+sidebar_position: 8
+title: Additional Resources
+---
+
+Here's a list of resource links that could be helpful during development while using the Stream Chat React Native SDK library.
```

Comment:
Better to describe the goal here. "Stream Chat React Native uses a number of technologies, libraries, and techniques to deliver an easy to use Chat SDK. The scope of this documentation does not cover all of these, as such we have compiled a list of resources that can be helpful during development.